agent/functional_tests/util/utils_unix.go

```diff
@@ -238,3 +243,13 @@ func (agent *TestAgent) getBindMounts() []string {
 func (agent *TestAgent) Cleanup() {
 	agent.platformIndependentCleanup()
 }
+
+// AttributesToMap transforms a list of key, value attributes to return a map
+func AttributesToMap(attributes []*ecs.Attribute) map[string]string {
+	attributeMap := make(map[string]string)
+	attribs := attributes
```

Comment:
I think you can just get rid of this line by using `for _, attribute := range attributes`